src/Examples/Flowsheets/power_generation/rsofc/cpu.py

```diff
@@ -0,0 +1,720 @@
+        self.inlet.add(self.inlet_temperature, "temperature")
+        self.inlet.add(self.inlet_pressure, "pressure")
+        self.inlet.add(self.inlet_mole_frac_comp, "mole_frac_comp")
+
+        self.pureco2.add(self.pureco2_flow_mol, "flow_mol")
+        self.pureco2.add(self.pureco2_temperature, "temperature")
+        self.pureco2.add(self.pureco2_pressure, "pressure")
+        self.pureco2.add(self.pureco2_mole_frac_comp, "mole_frac_comp")
+
+        self.water.add(self.water_flow_mol, "flow_mol")
+        self.water.add(self.water_temperature, "temperature")
+        self.water.add(self.water_pressure, "pressure")
+        self.water.add(self.water_mole_frac_comp, "mole_frac_comp")
+
+        self.vent.add(self.vent_flow_mol, "flow_mol")
+        self.vent.add(self.vent_temperature, "temperature")
+        self.vent.add(self.vent_pressure, "pressure")
+        self.vent.add(self.vent_mole_frac_comp, "mole_frac_comp")
+
+    def make_vars(self):
+        """ This section is for creating all the vars for this model.
+            There are 1 inlet and 3 outlet streams.
+            These streams are names as inlet, pureco2, water, vent
+            For each of these streams, the following variables are defined:
+            (1) Total mole flow [mol/s]: [stream_name]_flow_mol
+            (2) Component molar fraction: [stream_name]_mole_frac_comp
+            (3) Component mole flows [mol/s]: [stream_name]_flow_mol_comp
+            (4) Temperature [K]: [stream_name]_temperature
+            (5) Pressure [Pa]: [stream_name]_pressure
+
+        """
+        # units declaration for vars
+        flow_units = pyunits.mol / pyunits.s
+        pressure_units = pyunits.Pa
+        temperature_units = pyunits.K
+        # heat_duty_units = pyunits.J/pyunits.s
+
+        # Total mole flow [mol/s]
+        self.inlet_flow_mol = Var(
+            self.flowsheet().config.time,
+            initialize=3600,
+            # bounds=(1367.08, 5000),
+            units=flow_units,
+            doc="Total inlet mole flow [mol/s]",
+        )
+        self.pureco2_flow_mol = Var(
+            self.flowsheet().config.time,
+            initialize=900,
+            units=flow_units,
+            doc="PureCO2 outlet stream mole flow [mol/s]",
+        )
+        self.water_flow_mol = Var(
+            self.flowsheet().config.time,
+            initialize=300,
+            units=flow_units,
+            doc="Water outlet stream mole flow [mol/s]",
+        )
+        self.vent_flow_mol = Var(
+            self.flowsheet().config.time,
+            initialize=2400,
+            units=flow_units,
+            doc="Vent gas stream mole flow [mol/s]",
+        )
+
+        # Component molar fractions
+        self.inlet_mole_frac_comp = Var(
+            self.flowsheet().config.time,
+            self.component_list,
+            initialize=1 / len(self.component_list),
+            doc="Inlet stream: Component mole fraction",
```

Comment:
Should assign units here (`pyunits.dimensionless`). This applies to the following mole frac vars as well.